azure/terraform/registrieswebhooks.rego

```diff
@@ -34,16 +34,15 @@ acr_webhooks = false {
     azure_issue["acr_webhooks"]
 }
 
-acr_webhooks_err = "azurerm_container_registry_webhook property 'service_uri' need to be exist. Its missing from the resource." {
-    azure_attribute_absence["acr_webhooks"]
-} else = "Azure ACR currently does not have HTTPS protocol enabled for webhook" {
-    azure_issue["acr_webhooks"]
+acr_webhooks_miss_err = "azurerm_container_registry_webhook property 'service_uri' need to be exist. Its missing from the resource." {
```

Comment:
@shahinshirvani any reason not to include this in a else block with `acr_webhooks_err `?

Reply:
@rezoan fixed
at first I didn't know about this kind of new error message